.azure/regression-pipeline.yaml

```diff
@@ -61,7 +61,9 @@ jobs:
       test_case: 'mirrormaker/**/*ST,kafka/dynamicconfiguration/**/*ST'
       groups: 'regression'
       cluster_operator_install_type: 'bundle'
-      timeout: 360
+      timeout:
```

Comment:
@see-quick Are you sure this works? This looks like invalid YAML without the timeout.